### PR TITLE
Add PRESENTASJON_VARBEID in ontology

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

**Review by Eirik-Eide-Saur — APPROVED** [review]

**Review by brinxmat — DISMISSED** [review]

File: docs/cristin.ttl [hunk 1]
```diff
@@ -72,6 +72,37 @@
     and they have contributed as "ILLUSTRATOR" affiliated with InstB for the same publication.
     """ .
 
+:TYPE_MEDIEBIDRAG_Class
+    a            :CristinClass ;
+    rdfs:comment """
```
Comment (kariht) [inline docs/cristin.ttl:77]: har følgende kategorier:
DOKUMENTAR
INTERVJU
PROGDELTAGELSE
PROGLEDELSE

i ett av følgende medium:
AVIS
TV
INTERNETT
RADIO
TIDSSKRIFT
FAGBLAD

File: docs/cristin.ttl [hunk 2]
```diff
@@ -72,6 +72,37 @@
...
+    The structure of TYPE_MEDIEBIDRAG. UNKNOWN meaning
+            """ .
+
+:PRESENTASJON_VARBEID_Class
+    a            :CristinClass ;
+    rdfs:comment """
+    The structure of PRESENTASJON_VARBEID. UNKNOWN meaning
```
Comment (kariht) [inline docs/cristin.ttl:84]: Information about what project the publication is a part of

**Review by Eirik-Eide-Saur — DISMISSED** [review]

File: docs/cristin.ttl (hunk 1 above)
Comment (axthosarouris) [reply docs/cristin.ttl:79]: In the export we receive the following object:
 "type_mediebidrag": {
    "varbeidlopenr": 965431
  }
and it seems that it is always equal to respective Cristin result ID, i.e., it is always  a self reference and therefore we can ignore it. Is my assumption correct?

File: docs/cristin.ttl [hunk 3]
```diff
@@ -523,6 +575,60 @@
...
+    rdfs:domain  :Result ;
+    rdfs:comment """A Cristin Result will have such a field only when it's varbeidunderkatkode has one of the following values:
+	PROGLEDELSE, PROGDELTAGELSE, DOKUMENTAR, INTERVJU""" ;
+    :isignored   true .
+
+:varbeidlopenr
+    a            :cristinfield ;
+    rdfs:range   xsd:integer ;
+    rdfs:domain  :TYPE_MEDIEBIDRAG_Class ;
+    rdfs:comment """The Cristin Id of the respective :Result. It is a self reference.""" ;
```
Comment (axthosarouris) [inline docs/cristin.ttl:591]: @kariht  Is this correct?

**Pull request merged**